### PR TITLE
Tooltip: Style Variant "Fixed"

### PR DESCRIPTION
**Pull request by nickwmurphy**
### Summary
- New `styleVariants` prop with `'fixed'` as an option
- Tooltip stays open when hovering over the tooltip element itself
- `delay` prop is now `delayShow`
- Sandbox cleanup

### CODE REVIEW

File: tooltip/src/css-overrides/fixedStyleOverrides.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.tooltipContent {
+  position: fixed;
+}
```
Comment (dancingshell) [inline tooltip/src/css-overrides/fixedStyleOverrides.scss]: yay style variants!

nb: I think in the other particles we had named the file after the variant `css-overrides/fixed.scss`

File: tooltip/src/Tooltip.jsx [hunk 2]
```diff
@@ -38,13 +48,18 @@ export default class TooltipWrapper extends React.Component {
...
+    styleVariants: PropTypes.arrayOf(PropTypes.string)
   };
 
   constructor(props) {
     super(props);
 
-    this.styles = composeStyles(baseStyles, [...props.stylesheets]);
+    const { stylesheets, styleVariants } = props;
+
+    this.styles = composeStyles(baseStyles,
```
Comment (dancingshell) [inline tooltip/src/Tooltip.jsx]: also add this to `componentWillReceiveProps`?

Comment (nickwmurphy) [reply tooltip/src/Tooltip.jsx]: :+1: done

File: tooltip/src/Tooltip.jsx [hunk 3]
```diff
@@ -143,6 +170,10 @@ export default class TooltipWrapper extends React.Component {
     const { children, content, placement, popover } = this.props;
     const { show } = this.state;
 
+    if (!content) {
+      return null;
+    }
```
Comment (dancingshell) [inline tooltip/src/Tooltip.jsx]: only hide tooltip if no content, still render trigger

File: tooltip/src/Tooltip.scss [hunk 4]
```diff
@@ -7,6 +7,7 @@ $tooltipBackgroundColor: rgba(19, 31, 51, 0.9);
   box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
   border-radius: 2px;
   color: $white;
+  max-width: 400px;
```
Comment (dancingshell) [inline tooltip/src/Tooltip.scss]: max width should be 200px for tooltip, 250px for popover

**Pull request merged**